doc/manuals/orion-api.md

```diff
@@ -1637,6 +1757,23 @@ Content-Type is `application/json`
 
 The request payload is the new attribute value.
 
+_**Request URL parameters**_
+
+Those parameter are part of the URL request. They are mandatory. 
+
+| Parameter  | Type   | Description                          | Example    |
+|------------|--------|--------------------------------------|------------|
+| `entityId` | string | Id of the entity to be updated.      | `Room`     |
+| `attrName` | string | Name of the attribute to be updated. | `Location` |
+
+_**Request query parameters**_
+
+| Parameter  | Optional | Type   | Description                                                                                 | Example       |
+|------------|----------|--------|---------------------------------------------------------------------------------------------|---------------|
+| `type`     | ✓        | string | Entity type, to avoid ambiguity in case there are several entities with the same entity id. | `Room`        |
+
+_**Request headers**_
+
 * If the request payload MIME type is `application/json`, then the value of the attribute is set to
```

Comment:
I think this fragment (this bullet and the next one) should be moved to the "Request payload" section, as actually they are describing payload aspects.

Reply:
Fixed here https://github.com/telefonicaid/fiware-orion/commit/407453f9bbed05ad88353b101eef90c3ee273faa